### DISCUSSION TITLE
Welcome to streamwall Discussions!

### DISCUSSION THREAD

**Discussion by sayhiben** [chromakode/streamwall#148] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amarmashruwala** [comment]
Hello and thank you for this page and project, my name is Mash and I am about to rat your application this week. 

What are the formats it supports? 
Can it open multiple YouTube videos? 
How about multiple HLS streams? Or RTMP ? 
Thanks

**Comment by EpicMulletGuy** [comment]
Just installed v2 using the Windows binary, how do you configure this thing? V1 you loaded used npm and had config file was no issue. The new binary way no such config file to edit it loads what it loads and thats it am i missing something? Why not just have a simple "config" button on the page you add URL's for streams that lets you set grid size.
